imapsync/usr/lib/systemd/system/imapsync@.service

```diff
@@ -12,5 +12,5 @@ ExecStart=/usr/bin/imapsync --noauthmd5 --noreleasecheck --allowsizemismatch \
 --passfile2 /var/lib/nethserver/imapsync/vmail.pwd \
 --host1 ${REMOTEHOSTNAME} --port1 ${REMOTEPORT} ${SECURITY} --user1 ${REMOTEUSERNAME} \
 --passfile1 /var/lib/nethserver/imapsync/%i.pwd \
---exclude '^Public Folders$|^Trash$|^Deleted Items$|^Public${EXCLUDE}' \
+--exclude '^Public Folders|^Trash|^Deleted Items|^Public${EXCLUDE}' \
 --logdir /var/log/imapsync
```

Comment:
There are multiple log files under that directory, one for each run. Can we change the configuration to have one log for each user?

    Log file is /var/log/imapsync/2020_10_20_07_52_58_admin@dpnet.nethesis.it*vmail.txt ( to change it, use --logfile filepath ; or use --nolog to turn off logging )

Reply:
Since the runs are started manually, I'd leave the current configuration: one log per run.
It's much simpler to read.

Reply:
This theoretically possible

```
Q. I need to log every output on a file named log.txt

R1. imapsync logs on a file by default, its name is given at the 
    beginning and the end of each run. This name is unique since
    it is compound of the current date and time and user2 value. 

R2. To change this default name, use --logfile log.txt

  imapsync ... --logfile log.txt
```

However I would like more inputs from @nrauso because a log of imapsync could be really long (you can imagine one line per synchronized messages), so for reading and finding an information it could be long. With the one log per sync, I assume an believe it could be easier.

Reply:
by the way we could display the log inside the UI also, some feed for thoughts